ci/scripts/run-codebuild/libs/executor.test.js

```diff
@@ -0,0 +1,86 @@
+import * as runCodebuild from './executor.js'
+import { CodeBuildRunner } from './codebuild-runner.js'
+
+import { mkdtempSync, rmdirSync, unlinkSync, writeFileSync } from 'fs'
+import { tmpdir } from 'os'
+import { join, sep } from 'path'
+
+const mockBuildInfo = {
+  buildComplete: false,
+  buildStatus: 'IN_PROGRESS',
+  buildNumber: '1',
+  projectName: 'mockProject',
+  logs: {
+    groupName: 'mockBuildLogGroupName',
+    streamName: 'mockBuildLogStreamName',
+    deepLink: 'mockBuildLogDeepLink'
+  },
+  startTime: Date.parse('2021-07-05T10:00:00Z'),
+  endTime: Date.parse('2021-07-05T10:10:00Z')
+}
+
+jest.mock('./codebuild-runner.js', () => {
+  return {
+    CodeBuildRunner: jest.fn().mockImplementation(() => {
+      return {
+        getBuildInfo: jest.fn().mockReturnValue(mockBuildInfo),
```

Comment:
Are we testing the error case where this is used?

Reply:
I'm not, there was so much going on I decided to leave this case out to be honest

Reply:
Fair enough - we can always revisit in the future